includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -73,7 +74,14 @@ public function admin_notices() {
 		}
 
 		foreach ( (array) $this->notices as $notice_key => $notice ) {
-			echo '<div class="' . esc_attr( $notice['class'] ) . '" style="position:relative;">';
+			$has_actions = count( $notice['actions'] ) > 0;
```

Comment:
Just to be safe and prevent warnings, I think we should make sure that `$notice['actions']` is defined and countable.
```suggestion
			$has_actions = count( $notice['actions'] ?? [] ) > 0;
			// Or even something like this:
			$notice_actions = $notice['actions'] ?? [];
			$has_actions   = is_array( $notice_actions ) && [] !== $notice_actions;
```

Reply:
Alright. Refactored it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4669/commits/d56e1ea6699e0f2f5b2f3b82a6c5e91eab30776b